jobs/settings.py

```diff
@@ -229,3 +231,30 @@
     "General Configuration Service": ("SITE_NAME", "SITE_DESCRIPTION"),
     "Jobs Configuration Service": ("JOBS_URL",),
 }
+
+# Celery settings
+CELERY_HIGH_QUEUE_NAME = "high_priority"
+CELERY_LOW_QUEUE_NAME = "low_priority"
+CELERY_REDIRECT_STDOUTS_LEVEL = env("CELERY_REDIRECT_STDOUTS_LEVEL", default="DEBUG")
+CELERY_BROKER_URL = env("CELERY_BROKER_URL", default="redis://127.0.0.1:6379/0")
```

Comment:
the default should be `redis://redis:6379/0` Since you are developing with docker-compose, redis will be in `redis` host because the docker-compose network.
And also is a good practice to create multiple env vars like, `BROKER_HOST`, `BROKER_PORT`, `BROKER_DB` and then create `CELERY_BROKER_URL = f"{{BROKER_PROTOCOL}://{BROKER_HOST}:{BROKER_PORT}/{BROKER_DB}}"`

Reply:
Done!